src/sentry/search/events/fields.py

```diff
@@ -1189,6 +1194,8 @@ def get_type(self, value: str) -> str:
                 expression = field.get_expression(None)
                 if expression == value:
                     return field.result_type
+            else:
+                raise AssertionError(f"unreachable: {value}")
```

Comment:
there's a slight behaviour change here -- however a `list` value can't be passed into the baseclass as it would always crash so I think (?) this is fine